src/api-server.js

```diff
@@ -164,6 +173,19 @@ app.route('/login/').get((req,res) => {
 	}
 })
 
+function logfile(logContent, filePath){
+	try {
+		fs.mkdirSync(path.dirname(filePath), { recursive: true });
+		fs.writeFileSync(filePath, logContent, { flag: 'a+' });
+		console.log('Log file and folder created successfully');
+	} catch (error) {
+		if (error.code === 'EEXIST') {
```

Comment:
Please only do this error check when creating the folder/log file (which should happen only once, on server init).